### ISSUE TITLE
Drive set as Hidden + System item?

### ISSUE THREAD

**Issue by DavidBevi** [omeryanar/FileExplorer#61]
**Issue**: my `D:` drive (internal) is not shown unless I enable both "Show Hidden Items" and "Show System Items".
**Expected behavior**: each drive/partition should be shown, like `C:` does (am I missing an option to do it already?)

**Workaround**: `D:` is always accessible by typing "D:" into the address bar, even when the drive is hidden.
![EF_hidden_D_drive](https://github.com/omeryanar/FileExplorer/assets/61429850/220272a6-c99e-4c46-b009-29cb6e8e78e3)

**Comment by DavidBevi** [comment]
Also, what is the "**Total**" in the bottom bar counting?

**Comment by omeryanar** [comment]
This is odd. Drives are initialized with FromDriveInfo method (Model/FileModel.cs) and there is no code in this method that sets IsHidden or IsSystem fields. The only thing I can think of is that the drive was mounted while FileExplorer was already running. Is that the case? In addtion, do you have a chance to debug and show me how these fields are set? You can set a breakpoint at GetDrives method inside Helpers/FileSystemHelper.cs and follow from there.

**Comment by omeryanar** [comment]
> Also, what is the "**Total**" in the bottom bar counting?

It shows how many folders or files are inside the current folder. This can be confusing in the "Content View" because if you expand a folder, its subfolders and files are also included in the count.

**Comment by DavidBevi** [comment]
> [...] the drive was mounted while FileExplorer was already running.
PC mounts it while booting, does FE load before that? I'll try disabling FE's auto load.

> [...] do you have a chance to debug and show me how these fields are set? 
Maybe, but I don't know. I just started learning C. I'll try to do it monday.

**Comment by omeryanar** [comment]
> 
> PC mounts it while booting, does FE load before that? I'll try disabling FE's auto load + load it later to rule out that option and see if anything changes.
> 

No, normally FileExplorer runs after booting. However, if the problem goes away when you exit and restart FileExplorer (from the system tray icon), it must be related to an issue with booting.

> 
> Maybe, but I don't know. I just started learning C. I'll try to do it tomorrow, feel free to help me if you want quick answers, but I want to learn it anyway so you'll get it eventually 😉

You can use dnSpy: https://github.com/dnSpy/dnSpy 
But it may seem complicated if you've never done it before

**Comment by DavidBevi** [comment]
1. The problem persists after I exit and restart FileExplorer from sys tray.

2. Thanks for the tip. Yes, debugging seems daunting, that's why I procrastinated. I'm checking out dnSpy rn
